File: java-course/src/test/java/com/example/selenium/spree/pages/ProductPage.java
package com.example.selenium.spree.pages;

import org.openqa.selenium.remote.RemoteWebDriver;
import org.testng.Assert;

public class ProductPage  extends BasePage {
    public static final String URL = "https://spreecommerce-demo.herokuapp.com/products/spree-tote";

    public static final String TITLE = "Spree Tote - Spree Demo Site";

    public ProductPage(RemoteWebDriver webDriver) {
        super(webDriver, TITLE, URL);
    }
}

File: python-course/com/example/selenium/spree/pages/ProductPage.py
from com.example.selenium.spree.pages.BasePage import BasePage


class ProductPage(BasePage):
    title = "Spree Tote - Spree Demo Site"
    url = "https://spreecommerce-demo.herokuapp.com/products/spree-tote"

    def __init__(self, test):
        BasePage.__init__(self, test, self.title, self.url)
        self.test = test
